frontend/dispatcher.lua

```diff
@@ -33,8 +33,10 @@ local settingsList = {
     -- Device settings
     show_frontlight_dialog = { category="none", event="ShowFlDialog", title=_("Show frontlight dialog"), device=true, condition=Device:hasFrontlight(),},
     toggle_frontlight = { category="none", event="ToggleFrontlight", title=_("Toggle frontlight"), device=true, condition=Device:hasFrontlight(),},
+    set_frontlight = { category="incrementalnumber", event="SetFlIntensity", min=1, max=Device:getPowerDevice().fl_max, title=_("Set frontlight brightness"), device=true, condition=Device:hasFrontlight(),},
     increase_frontlight = { category="incrementalnumber", event="IncreaseFlIntensity", min=1, max=Device:getPowerDevice().fl_max, title=_("Increase frontlight brightness"), device=true, condition=Device:hasFrontlight(),},
     decrease_frontlight = { category="incrementalnumber", event="DecreaseFlIntensity", min=1, max=Device:getPowerDevice().fl_max, title=_("Decrease frontlight brightness"), device=true, condition=Device:hasFrontlight(),},
+    set_frontlight_warmth = { category="incrementalnumber", event="SetFlWarmth", min=1, max=Device:getPowerDevice().fl_warmth_max, title=_("Set frontlight warmth"), device=true, condition=Device:hasNaturalLight(),},
```

Comment:
Same as above.
Also max for the warmth should be 100 as powerd takes care of scaling the number in the backend, the frontend always sees 0 .. 100
This will fix your issue with the Clara.